engine/tester/config_parse.c

```diff
@@ -996,18 +1000,197 @@ get_test_attrs(xmlNodePtr node, test_attrs *attrs)
     return 0;
 }
 
+/**
+ * Get run template.
+ *
+ * @param node        XML node with script call description
+ * @param session     Location for session description
+ *
+ * @return Status code.
+ */
+static te_errno
+get_run_template(xmlNodePtr node, test_session *session)
```

Comment:
It should not be a dedicated function. It should reuse run_item parser with some limitations for template.
For example, it cannot be a package or session inside, either script or just args-only.